oak_functions/sdk/oak_functions/src/lib.rs

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! SDK functionality that provides idiomatic Rust wrappers around the
+//! underlying Oak functions platform functionality.
+
+use log::{debug, error};
+use oak_functions_abi::OakStatus;
+
+/// Reads and returns the user request.
+///
+/// This function is idempotent. Multiple call to this function all return the same value.
+pub fn read_request_body() -> Result<Vec<u8>, OakStatus> {
```

Comment:
Should this function be renamed to `read_request` (as comment calls it)
And `read_request` to `read_request_util` ?

Reply:
Yes. Done.